translations/monero-core_de.ts

```diff
@@ -140,19 +140,19 @@
     <message>
         <location filename="../components/DaemonConsole.qml" line="108"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Schlie&szlig;en</translation>
```

Comment:
as @JamesCullum pointed out, build fails on this line because of the htmlencoding.